test/include/util/catalog_test_util.h

```diff
@@ -33,5 +35,10 @@ struct CatalogTestUtil {
     }
     return catalog::Schema(columns);
   }
+
+  // Define contants here for when you need to fake having a catalog in tests. These values may change as the catalog
+  // comes in, and tests should be modified accordingly.
+  static constexpr catalog::db_oid_t generic_db_oid = catalog::db_oid_t(101);
```

Comment:
nit: maybe put the word `test` in these names too, just to make sure.

Reply:
In theory the `src` folder shouldn't be able to see it since it's in the test folder, but I can do that.